data/scripting/win_conditions/win_condition_functions.lua

```diff
@@ -349,6 +385,15 @@ end
 --    the message window pops up ever hour, 30, 20 & 10 minutes before the game ends.
 --
 --    :arg max_time:    The time maximum game time in minutes
+--    :type max_time: :class:`integer`
+--    :arg remaining_time: The remaining time until game ends. On first call
+--                         this is equal to **max_time**.
```

Comment:
I think this should use code markup?

Reply:
That's what i meant with 'referring to a functions argument'. In the description of a function the arguments are shown _cursive_ while in the list of arguments this is __bold__. Which markup should we use to refer to a functions argument in plain text descriptions?
I am in favor of _cursive_, although i used bold here... Imho this should be either cursive or bold to have visual correspondence.

Reply:
+1 for consistence. One of these two font styles (**bold**/*italics*) should be reserved to emphasise important pieces of text, the other for parameter names. 
IMHO *italics* seem less importunate / more reader-friendly than **bold** print, and should therefore be preferred for the first purpose, and **bold* letters used for the variables.

Reply:
So this is fine...